services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/diagnostic_reports/diagnostics/__tests__/studentNameOrTooltip.test.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import * as $ from 'jquery'
+
+import StudentNameOrTooltip from '../studentNameOrTooltip'
+
+describe('StudentNameOrTooltip component', () => {
+  it('should render if the name is long enough to warrant a tooltip', () => {
+    const wrapper = mount(<StudentNameOrTooltip
+      name="Gabriel De La Concordia Garcia Gabriel De La Concordia Garcia Gabriel De La Concordia Garcia"
```

Comment:
one of my best friends just recommended that I read _Love in the Time of Cholera_  <3